### PR TITLE
RHDEVDOCS 5978 startingDeadlineSeconds parameter

### PR DESCRIPTION
**Pull request by mramendi**
Version(s):

please cp to pipelines-docs-1.15

Issue:

RHDEVDOCS 5978

Link to docs preview:

https://77607--ocpdocs-pr.netlify.app/openshift-pipelines/latest/install_config/customizing-configurations-in-the-tektonconfig-cr.html#default-pruner-configuration_customizing-configurations-in-the-tektonconfig-cr

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by mramendi** [comment]
https://issues.redhat.com/browse/RHDEVDOCS-5978

**Review by jkandasa — APPROVED** [review]
LGTM

File: modules/op-default-pruner-configuration.adoc [hunk 1]
```diff
@@ -49,6 +50,10 @@ If set to `true`, the value for the `keep` parameter is calculated separately fo
 
 |`keep-since`
 |The maximum time for which to keep resources, in minutes. For example, to retain resources which were created not more than five days ago, set `keep-since` to `7200`.
+
+|`startingDeadlineSeconds`
+|This parameter is optional. If the pruner job is not stated at the scheduled time for any reason, this setting configures the maximum time, in seconds, in which the job can still be started. If the job is not started within the specified time, {pipelines-shortname} considers this job failed and starts the pruner at the next scheduled time. If you do not specify this parameter and the pruner job does not start at the scheduled time, {pipelines-shortname} attempts to start the job at any later time possible.
```
Comment (piyush-garg) [inline modules/op-default-pruner-configuration.adoc:55]: typo - started

**Review by ppitonak — APPROVED** [review]

**Comment by piyush-garg** [comment]
lgtm

**Comment by dfitzmau** [comment]
LGTM 

/remove-label peer-review-in-progress

/label peer-review-done

**Review by skopacz1 — APPROVED** [review]
Merge review LGTM

**Pull request merged**